hackathon_site/dashboard/frontend/src/slices/order/orderSlice.ts

```diff
@@ -89,6 +112,29 @@ const orderSlice = createSlice({
                 payload?.message ||
                 "There was a problem retrieving orders. If this continues please contact hackathon organizers.";
         });
+
+        builder.addCase(updateOrder.pending, (state) => {
+            state.isLoading = true;
+            state.error = null;
```

Comment:
watch out what ur setting as loading and error, these states are for a different thunk (getTeamOrders thunk) so best not to interfere there, create new states for loading & error for cancelling